### ISSUE TITLE
What `Illegal instruction:` means?

### ISSUE THREAD

**Issue by o-zaporozhchenko** [nicklockwood/SwiftFormat#331]
What `Illegal instruction:` means?
I get: 
```
swiftformat --indentcase true .
running swiftformat...
Illegal instruction: 4
```

**Comment by nicklockwood** [comment]
@o-zaporozhchenko it means swiftformat crashed (it's not supposed to do that).

That probably means it choked on one of your source files - I'd appreciate if you could work out which file it was and send me a copy :-)

**Comment by nicklockwood** [comment]
@o-zaporozhchenko out of curiosity, what OS version are you running on?

**Comment by nicklockwood** [comment]
@o-zaporozhchenko if you run swiftformat with the --verbose option, it might help to isolate the file that crashed.

**Comment by o-zaporozhchenko** [comment]
@nicklockwood 10.13.6 High Sierra

**Comment by o-zaporozhchenko** [comment]
@nicklockwood so i did investigation:

Minimal piece code that breaks `swiftformat` is:



import Foundation

```


import Foundation

struct SomeModel {
    
    init(uid: String

//        someCommentedProperty:[String:Any?]?,
//        value:Int?,

        ) {
        
        self.uid = uid
    }
}


```

I suppose, commented multiple properties cause the problem there

```
//        someCommentedProperty:[String:Any?]?,
//        value:Int?,

```

**Comment by o-zaporozhchenko** [comment]
@nicklockwood Hope u can fix it. Also, as i see version in homebrew is lower than on github source page. 
Would be nice to update it :)
0.35.9 on github and 0.35.7 installed via homebrew

**Comment by o-zaporozhchenko** [comment]
@nicklockwood 
oh, i found that `brew update` makes
`touch: /usr/local/Homebrew/.git/FETCH_HEAD: Permission denied
` and others problems
and 
`sudo brew update` is forbidden now
xD

**Comment by nicklockwood** [comment]
@o-zaporozhchenko this crash is fixed in version 0.35.10
